src/core/sig_state.ml

```diff
@@ -119,16 +119,14 @@ let of_sign : Sign.t -> sig_state = fun signature ->
     are allowed in left-hand side of rewrite rules (only) iff [~prt] is true.
     {!constructor:Term.expo.Privat} symbols are allowed iff [~prv]
     is [true]. *)
-let find_sym : prt:bool -> prv:bool -> sig_state -> p_qident -> sym =
+let find_sym : prt:bool -> prv:bool -> sig_state -> qident loc -> sym =
   fun ~prt ~prv st {elt = (mp, s); pos} ->
-  let pp_uid = Parsing.LpLexer.pp_uid in
-  let pp_path = Lplib.List.pp pp_uid "." in
   let s =
     match mp with
     | [] -> (* Symbol in scope. *)
         begin
           try fst (StrMap.find s st.in_scope)
-          with Not_found -> fatal pos "Unknown object %a." pp_uid s
+          with Not_found -> fatal pos "Unknown object %s." s
```

Comment:
Ah, this reverting many changes I made recently in order to print identifiers correctly... Identifiers should not be printed as strings unless we decide to keep brackets in escaped identifier. See https://github.com/Deducteam/lambdapi/pull/627/files#r603836591.

Reply:
Yeah, I realise that. It would motivate going back to the previous strategy and have strings with booleans to say whether it should be escaped or not.

Reply:
No need for a boolean. Escape.is_escaped tells you whether an identifier is escaped or not.

Reply:
Whether it is escaped, but not whether it should be escaped, right?

Reply:
As explained in https://github.com/Deducteam/lambdapi/pull/627/files#r603836591, the only places where we have to take care of this is with filenames and when adding suffixes or prefixes to existing identifiers.

Reply:
I can take care of this if you want.

Reply:
You can :)